uk.co.compucorp.civicrm.hrsampledata/CRM/HRSampleData/CSVHandler.php

```diff
@@ -86,6 +91,24 @@ protected function callAPI($entity, $action, $params) {
   }
 
   /**
+   * Deletes entity record from the database based on
+   * search criteria
+   *
+   * @param string $entity
+   * @param array $searchParams
+   */
+  protected function deleteRecord($entity, $searchParams) {
+    $searchParams['options'] = ['limit' => 0];
```

Comment:
from my quick testing I found that you cannot delete more than one record with the API at time , so I guess it make sense to keep the limit = 0